OJP/OJP_Availability.xsd

```diff
@@ -0,0 +1,426 @@
+				<xs:annotation>
+					<xs:documentation>Location where passenger intends to leave the service.</xs:documentation>
+				</xs:annotation>
+			</xs:element>
+			<xs:group ref="ServiceGroup" minOccurs="0">
+				<xs:annotation>
+					<xs:documentation>Properties of a service (line and direction) by which the availability is being checked: attributes of ContinuousServiceStructure or DatedJourneyStructure.</xs:documentation>
+				</xs:annotation>
+			</xs:group>
+			<xs:element name="Extension" type="xs:anyType" minOccurs="0"/>
+		</xs:sequence>
+	</xs:complexType>
+	<xs:complexType name="BookingUserStructure">
+		<xs:annotation>
+			<xs:documentation>Passenger(s) for whom the service needs to be booked.</xs:documentation>
+		</xs:annotation>
+		<xs:sequence>
+			<xs:element name="Age" type="xs:nonNegativeInteger" minOccurs="0">
+				<xs:annotation>
+					<xs:documentation>Age of the passenger on the day of travel.</xs:documentation>
+				</xs:annotation>
+			</xs:element>
+			<xs:group ref="PassengerProfileGroup"/>
+			<xs:group ref="PassengerRequirementsGroup"/>
+		</xs:sequence>
+	</xs:complexType>
+	<xs:complexType name="ProvisionedCallAtPlaceStructure">
+		<xs:annotation>
+			<xs:documentation>Provisioned vehicle call at a general location.</xs:documentation>
+		</xs:annotation>
+		<xs:sequence>
+			<xs:group ref="DatedJourneyRefGroup" minOccurs="0"/>
+			<xs:element name="CallPlace" type="PlaceRefStructure">
+				<xs:annotation>
+					<xs:documentation>More general location for a call than stop points. May be used with flexible services or "Area Dial-A-Ride".</xs:documentation>
+				</xs:annotation>
+			</xs:element>
+			<xs:element name="ServiceArrival" type="ServiceArrivalStructure" minOccurs="0">
+				<xs:annotation>
+					<xs:documentation>Arrival times of the service at this stop.</xs:documentation>
+				</xs:annotation>
+			</xs:element>
+			<xs:element name="ServiceDeparture" type="ServiceDepartureStructure" minOccurs="0">
+				<xs:annotation>
+					<xs:documentation>Departure times of the service at this stop.</xs:documentation>
+				</xs:annotation>
+			</xs:element>
+			<xs:group ref="StopCallStatusGroup" minOccurs="0"/>
+		</xs:sequence>
+		<!-- Adapted from Trias_JourneySupport.xsd -->
+	</xs:complexType>
+	<xs:group name="MinMaxTimeBandGroup">
+		<xs:annotation>
+			<xs:documentation>Earliest possible departure time from a start location and latest possible arrival at a different destination location.</xs:documentation>
+		</xs:annotation>
+		<xs:sequence>
+			<xs:element name="EarliestDepartureTime" type="xs:dateTime" minOccurs="0"/>
+			<xs:element name="LatestArrivalTime" type="xs:dateTime" minOccurs="0"/>
+		</xs:sequence>
+	</xs:group>
+	<xs:annotation>
+		<xs:documentation>========================================== From Trias_JourneySupport.xsd ==========================================</xs:documentation>
+	</xs:annotation>
+	<xs:group name="PassengerProfileGroup">
```

Comment:
Question / doubt: will all of the profiles ever bee taken into account? How exactly?

Reply:
@trurlurl Isn't that from your input?

Reply:
Yes, but it's taken from TRIAS, I thought perfect alignment with TRIAS was intended. Now, after the discussions I've heard in Köln, in particular about the seach options and preferences, I have doubts. Shouldn't determining the appropriateness for, e.g., *WalkingFrame* be done in an "intermediate layer", similarly as for  TripRequest? Do we want or have to support the same profiles as TRIAS? And who knows how are the criteria defined / operationalized?

Reply:
ok